utilities/ft_hastoolbox.m

```diff
@@ -162,6 +162,7 @@
   'MVPA-LIGHT'    'see https://github.com/treder/MVPA-Light'
   'XDF'           'see https://github.com/xdf-modules/xdf-Matlab'
   'MRTRIX'        'see https://mrtrix.org'
+  'BAYESFACTOR'   'see https://klabhub.github.io/bayesFactor'
```

Comment:
sorry, this change should not have been included in this PR. But it is harmless, so please ignore.